tool-plugins/intellij/src/main/java/io/ballerina/plugins/idea/runconfig/application/BallerinaApplicationRunningState.java

```diff
@@ -67,18 +68,36 @@ public void startNotified(ProcessEvent event) {
 
     @Override
     protected BallerinaExecutor patchExecutor(@NotNull BallerinaExecutor executor) throws ExecutionException {
-        RunConfigurationKind kind = getConfiguration().getRunKind();
+        // RunConfigurationKind kind = getConfiguration().getRunKind();
```

Comment:
Lets remove the commented codes as well.

Reply:
Removed